server/vulnerabilities/oval/mappers.go

```diff
@@ -89,12 +89,16 @@ func mapPackageState(sta oval_input.DpkgStateXML) ([]oval_parsed.ObjectStateEvrS
 }
 
 func mapPackageObject(obj oval_input.DpkgObjectXML, vars map[string]oval_input.ConstantVariableXML) ([]string, error) {
+	if obj.Name.Value != "" {
+		return []string{obj.Name.Value}, nil
+	}
+
+	var r []string
 	variable, ok := vars[obj.Name.VarRef]
 	if !ok {
-		return nil, fmt.Errorf("variable not found %s", obj.Name.VarRef)
+		return nil, fmt.Errorf("OVAL parse: variable not found %s", obj.Name.VarRef)
```

Comment:
Maybe we can just do `return fmt.Errorf("oval parse: %w", err)` in:
https://github.com/fleetdm/fleet/blob/fb6a719fa1cf414b984dc97731d1d262c8044181/server/vulnerabilities/oval/parser.go#L24

Reply:
Actually we can't do `".. %w", err)` because no error per se is returned - I'll change the capitalization though (since if I remember correctly that's a best practice in Go)

Reply:
I would revert this change and only wrap the error as described with `%w` in `parseDefinitions`.

Reply:
Done